pkg/virt-handler/device-manager/socket_device.go

```diff
@@ -258,13 +259,10 @@ func (dpi *SocketDevicePlugin) healthCheck() error {
 	}
 	defer watcher.Close()
 
-	// This way we don't have to mount /dev from the node
```

Comment:
nit - unrelated comment removal?

Reply:
It was a leftover from a copy-paste from another device plugin. We don't use any /dev with the socket plugin